client/src/views/logs.tsx

```diff
@@ -9,6 +9,8 @@ import InputFilter from '../components/inputFilter';
 import Loading from '../components/loading';
 import api from '../services/api';
 import { Pod } from '../utils/types';
+import Button from '../components/button';
+import LogsSvg from '../art/logsSvg';
```

Comment:
I don't see this file in the PR. Did you perhaps forget to include it?

Reply:
No, those should already exist in this repository.